bugzilla/base.py

```diff
@@ -1489,7 +1489,7 @@ def s(key, val, convert=None):
                 val = convert(val)
             ret[key] = val
 
-        def add_dict(key, add, remove, _set=None, convert=None):
```

Comment:
If I'm not mistaken, all uses of `convert` inside of the method were removed by #190. Now `pylint` complained about the argument not being used.

I checked with `grep` and this method never receives 5 arguments.

Reply:
Sure, makes sense :+1: . See #195.